device/celestica/x86_64-cel_e1031-r0/sonic_platform/common.py

```diff
@@ -47,7 +48,7 @@ def run_command(self, command):
         output = ""
         try:
             p = subprocess.Popen(
-                command, shell=True, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
+                split(quote(command)), stdout=subprocess.PIPE, stderr=subprocess.PIPE)

```

Comment:
>split [](http://example.com/codeflow?start=16&length=5)

This will break many command, like "echo hello".

Reply:
Fixed to use `split` only.